pkg/docker-ce/Dockerfile

```diff
@@ -1,3 +1,4 @@
+FROM linuxkit/vpnkit-forwarder:883de832c2c3cb72cd9b01e3f7bd788649e0f2c2 AS vpnkit
```

Comment:
I think its more logical to add this just before `FROM scratch` so we consistently do the `FROM linuxkit/alpine` at the top of files.